lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -1711,7 +1719,14 @@ def test_certificate_generation_whitelist_already_generated(self):
         }
         self.assertCertificatesGenerated(task_input, expected_results)
 
-    def test_certificate_generation_whitelisted_not_generated(self):
+    @ddt.data(
+        (CertificateStatuses.downloadable, 3),
+        (CertificateStatuses.generating, 3),
+        (CertificateStatuses.notpassing, 5),
```

Comment:
I have a confusion, if certificates are generated for 2 students with `notpassing` or `audit_passing`, then why the expected certs are 5? When 2 of them have already certs?